docs/en/stack/ml/df-analytics/ml-dfa-outlier-detection.asciidoc

```diff
@@ -196,40 +190,42 @@ the algorithm performance by using these values.
 
 
 [discrete]
-[[ecommerce-outliers]]
-== Detecting unusual behavior in the eCommerce data set
+[[weblogs-outliers]]
+== Detecting unusual behavior in the logs data set
 
 The goal of <<dfa-outlier-detection,{oldetection}>> is to find the most unusual
-documents in an index. Let's try to detect unusual customer behavior in the 
-{kibana-ref}/get-started.html#gs-get-data-into-kibana[eCommerce sample data set]. 
+documents in an index. Let's try to detect unusual behavior in the 
+{kibana-ref}/get-started.html#gs-get-data-into-kibana[data logs sample data set]. 
 
-. Verify that your environment is set up properly to use {ml-features}. 
-If the {es} {security-features} are enabled, you need a user that has authority
-to create and manage {dfanalytics-jobs}. See <<setup>>.
+. Verify that your environment is set up properly to use {ml-features}. If the 
+{es} {security-features} are enabled, you need a user that has authority to 
+create and manage {dfanalytics-jobs}. See <<setup>>.
 +
 --
-Since we'll be creating {transforms}, you also need
+Since we'll be creating {transforms}, you also need 
 `manage_data_frame_transforms` cluster privileges.
 --
 
 . Create a {transform} that generates an entity-centric index with numeric or
 boolean data to analyze.
 +
 --
-In this example, we'll use the eCommerce orders sample data and pivot the data
-such that we get a new index that contains a sales summary for each customer.
+In this example, we'll use the web logs sample data and pivot the data such 
+that we get a new index that contains a network usage summary for each client 
+IP.
 
-In particular, create a {transform} that calculates the sum of the products
-(`products.quantity`) and the sum of prices (`products.taxful_price`) in all of
-the orders, grouped by customer (`customer_full_name.keyword`). Also include a 
-value count aggregation, so that we know how many orders (`order_id`) exist for 
-each customer.
+In particular, create a {transform} that calculates the number of occasions when 
```

Comment:
This looks like a transform example embedded in an outlier detection example. It would be better to use a straightforward example where the end user is directly doing outlier detection and not any data processing operations.

Reply:
Yes, it is a transform example embedded in the outlier detection example. Unfortunately, the example wouldn't make sense if we don't transform the data into an entity-centric index where we can correlate the relevant values with the client IPs. The other reason why this example might be useful for users is that it shows how they can use transforms and DFA together which is expected as a common use case.